src/job_description_feature_extraction.py

```diff
@@ -0,0 +1,49 @@
+import string
+import operator
+from collections import Counter
+
+
+def get_one_hot_encoded_words(feature_to_extract):
+    """
+    get_feature_word_count(x) returns a list of tuples which consists of word:count
+    From those lists, the words appear in the list below ( word_list ) was hand picked
+    Fhey are the word which might have an impact on the salary
+    """
+    word_list = ['excellent', 'graduate', 'immediate', 'junior', 'urgent']
```

Comment:
we should think in more generic terms. apart from working on any data set and be more dynamic (i.e. use different cutoff, use top X words), the code would describe, how we came up with that list and we can argue about it, rather than pulling out words that "seem" relevant to you.